File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoLayoutFormFooter.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.MetaIdLayoutForm;
import com.neome.api.meta.base.Types.MetaIdVar;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class StudioDtoLayoutFormFooter extends StudioBase
{
  @Nullable
  public FieldDtoImage footerImage;

  @Nullable
  public Long footerImageHeight;

  @Nullable
  public MetaIdVar footerImageVarId;

  @Nullable
  public MetaIdLayoutForm formLayoutId;

  @Nullable
  public Boolean showSeparator;
}

File: app/src/main/java/com/neome/api/meta/base/dto/StudioDtoLayoutFormFooter.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.FieldDtoImage
import com.neome.api.meta.base.Types.MetaIdLayoutForm
import com.neome.api.meta.base.Types.MetaIdVar
import com.neome.api.meta.base.dto.StudioBase

interface StudioDtoLayoutFormFooter : StudioBase
{
  val footerImage: FieldDtoImage?
  val footerImageHeight: Long?
  val footerImageVarId: MetaIdVar?
  val formLayoutId: MetaIdLayoutForm?
  val showSeparator: Boolean?
}